src/js/xr/src/SceneManagerXR.js

```diff
@@ -502,10 +503,12 @@ const SceneContent = connect(
                     gltf={resources.controlsGltf}
                     mode={uiCurrent.value.controls}
                     hand={switchHand ? 'left' : 'right'}
+                    guiLocked={guiLocked}
```

Comment:
This could just be `locked={uiCurrent.context.locked}`

There's no need to also store `guiLocked` in the ui store.